public/js/specberus.js

```diff
@@ -192,35 +199,62 @@ jQuery.extend({
         attachCustomScroll();
         // endValidation();
     });
+    socket.on("finishedExtraction", function (data) {
+      $(".manual").toggle(true);
+      var processDocument = (data.processdocument && data.processDocument.indexOf('Process-20051014') > -1) ? "2005" : "2015";
+      $profile.val(data.profile);
+      $processDocument.find("label#2005").toggleClass("active", (processDocument === "2005"));
+      $processDocument.find("label#2015").toggleClass("active", (processDocument === "2015"));
+      $noRecTrack.prop('checked', !data.rectrack);
+      $informativeOnly.prop('checked', data.informative);
+      $validation.val('simple-validation');
+      var options = {
+                        "url"             : data.url
+                      , "profile"         : data.profile
+                      , "validation"      : 'simple-validation'
+                      , "noRecTrack"      : !data.rectrack || false
+                      , "informativeOnly" : data.informative || false
+                      , "echidnaReady"    : false
```

Comment:
Because, I think echidna is a special use case. This is not something that can be extracted from the document so the "default" run the echidna rules. The options can still be changed after the 1st run if necessary.